src/main/java/bio/terra/grammar/azure/SynapseVisitor.java

```diff
@@ -4,37 +4,38 @@
 import bio.terra.grammar.SQLParser;
 import bio.terra.model.DatasetModel;
 import bio.terra.service.resourcemanagement.azure.AzureStorageAccountResource.FolderType;
+import bio.terra.service.snapshotbuilder.query.TableNameGenerator;
 import java.util.Map;
 import java.util.Objects;
 
 public class SynapseVisitor extends DatasetAwareVisitor {
   private final String sourceDatasetDatasource;
 
+  private static final String azureTableName =
```

Comment:
We usually name static strings in all caps. You can see a few examples of this in [AzureSynapsePdao](https://github.com/DataBiosphere/jade-data-repo/blob/develop/src/main/java/bio/terra/service/filedata/azure/AzureSynapsePdao.java#L123).
```suggestion
  private static final String AZURE_TABLE_NAME_TEMPLATE =
```